### PR TITLE
Migrated static site to Hugo

### PR DESCRIPTION
**Pull request by peterzen**

### CODE REVIEW

**Comment by jholdstock** [comment]
Some observations:

- Adding a `.dockerignore` file to ignore `.git/` will reduce the size of the docker context from ~300MB to around ~150MB and reduce the chance of unnecessary rebuilds.
- On the press page is a link `Governing Decentralization: How On-Chain Voting Protocols Operate and Vary`. This works currently on decred.org, but does not work when I am hosting the hugo site locally. Same for some of the other links on this page.
- Translations work correctly if I run the old site locally. Changing language on the hugo site redirects me to decred.org rather than keeping me on my local instance.
- Contributors page - Community filter - Denys Zayets. The link for his company has changed from http://www.cryptobits.co/ to https://www.cryptobits.co/. The new link does not work.
- The exchanges page seems to be missing the blue page footer. Its present on https://decred.org/exchanges/
- "All downloads" link on the landing page does not work
- Looking at the footer on the landing page, the link "2018 Roadmap" has changed to simply "Roadmap"

**Comment by peterzen** [comment]
Thanks for reviewing, Jamie.  Some of the links not working had to do with the `baseURL` being set to decred.org - if you now run `bin/watch.sh` it'll set this to the local URL so there shouldn't be any absolute `href`'s anywhere.  I fixed the other problems as well, thanks for spotting these.

**Comment by peterzen** [comment]
(Squashed the branch)

**Comment by jholdstock** [comment]
I have a clean clone of the repo, checked out your PR. Running `bin/watch.sh` and visiting `localhost:1313` in browser. The observations above seem to be resolved but here is a bunch of new stuff I noticed:

- Translation does not work at all. Selecting a new lang on the homepage does nothing.
- Translation dropdown appears on `/exchanges`, `/downloads` and `/roadmap`. This isnt shown on current decred.org. Possibly affects other pages too.
- css on `/downloads` seems off. Link styles in both the intro paragraph and under Decrediton heading seem different. First img is hugo, second img is decred.org:
screenshot from 2019-01-17 10-39-03
screenshot from 2019-01-17 10-39-09
- Translation dropdown seems to include both `en` and `eng`. First img from hugo, second img from decred.org:
screenshot from 2019-01-17 10-50-02
screenshot from 2019-01-17 10-49-49
- Visiting `/brand` makes xhr requests to https://googleads.g.doubleclick.net and https://static.doubleclick.net. This does not happen on decred.org/brand.  Possibly affects other pages too. Seems like its due to youtube player which should only be used on the homepage.
- decred.org currently has short titles for each page. EG. `/roadmap` has page title `Decred - Roadmap`. Hugo is showing `Decred - Autonomous Digital Currency - Roadmap`. Shorter is preferred as it is more likely to show in browser tabs.
- Contributors list is outdated. eg. Clarissa Peereboom is appearing under Marketing on the hugo site, but not live decred.org.
- The page footer has become taller than decred.org. 285px vs 200px
- `/stakepools` has changed terminology from "Stake pool" to "Voting Service Provider". This is actually a step forward - this is the new preferred terminology. Just wanted to highlight in-case this change was included by mistake.
- Heading "3RD PARTY WALLETS" is missing from `/downloads`
- on `/press` in the "Releases" tab. The release text should have a large, italic, blue heading. The headings are only italic.
- Also on `/press` Releases tab - the dates are formatted like `June 5, 2018` instead of `2018-06-05`. Personally I think this is an improvement and we should also use this on the Coverage tab.
- `/press` releases tab - different formatting of the quotes, about decred, and contact section at the end of each release 
- `/contributors` page - Hovering on the raedah group logo shows a tooltip "raedah_group" instead of "Raedah Group LLC". Seems to affect all the company logos
- Contributors are not listed in correct alphabetical order. See Community contractors for the most stark example - Insaf is appearing before Bill

**Comment by peterzen** [comment]
Thanks again @jholdstock, all issues are sorted out.

**Comment by peterzen** [comment]
To #1 - the issue seemed to be that the clickable links inside the dropdowns didn't have padding and were thus difficult to hit (and the menu behaved as if the item was clicked, regardless.  Could you please verify if it now works correctly on your end?

**Comment by jholdstock** [comment]
Most of the above has been resolved. Some outstanding issues and a couple of new ones here:

- CSS on the download page still doesnt seem to match decred.org. The links in the intro paragraph are underlined, and visited links under "Decrediton" are coloured gray.
- Footer still seems to be taller than before. 220px vs 200px.
- `/press` - releases tab - The quotes, "About Decred" and "Media Contacts" sections are still not formatted like the old site. They should have a reduced font size and contained in a gray box.
- The portrait for contributor "Mariusz Szyma" has changed. (Community). There may be others.
- LinkedIn is missing for contributors "Macaulay Davies" (Dev) and "Elian Huesca" (community). There may be others.
- Twitter is missing for contributor "Mykolas Raižys" (dev). There may be others.

**Comment by peterzen** [comment]
The Press section inconsistencies stem from the markup that's generated from MD and is different from the current code - since linnutee is working on redesigning the section I think it's not worth tweaking the CSS at this point (it's rather messy).  The contributors on the original have just been changed again, hence the difference and conflict again.... @dajohi can you please hold off on new changes until this gets rolled out?

I'll fix the the others, thank you @jholdstock .

File: README.md [hunk 1]
```diff
@@ -110,30 +88,22 @@ Runs the docker image exposing the specified local port.
...
+    bin/watch.sh
     ```
 
-You should now be able to access the site at `http://localhost:8080`
+You should now be able to access the site at `http://localhost:1313`
 
 ## License
 
-decredweb is licensed under the liberal ISC License.
+decredweb is licensed under the liberal ISC License.
```
Comment (jholdstock) [inline README.md]: Not related to the work you are doing @peterzen, but while you are working on the README you might as well fix this

```suggestion
dcrweb is licensed under the liberal ISC License.
```

**Comment by peterzen** [comment]
image

**Pull request merged**